app/css/styles.css

```diff
@@ -0,0 +1,112 @@
+/*nav*/
+.trolley {
+    height: 50px;
+    margin: 5px;
+}
+
+/*book list*/
+h1 {
+    margin: 0;
+    padding: 20px 0;
+}
+
+.title {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.bookImage {
+    max-width: 100%;
+}
+
+/*filter column*/
+.searchAndFilterColumn {
+    padding: 0;
+}
+
+.filterColumn {
+    margin-top: 50px;
+}
+
+/*search bar*/
+.searchButton {
+    padding: 5px 5px;
+}
+
+.searchInput {
+    height: 32px;
+    outline: 0;
+}
+
+.searchForm {
+    padding: 0;
+}
+
+a:hover {
+    text-decoration: none;
+    color: black;
+}
+
+/*return to home*/
+.returnToHome {
+    padding: 20px;
+    margin-top: 50px;
+    margin-bottom: 5px;
```

Comment:
Needs a hover state pointer for whole button not just text

Reply:
Deleted button for this story, saved your comment for story 4